sessionsManager.js

```diff
@@ -329,7 +328,7 @@ module.exports.SOURCE_TYPE = SOURCE_TYPE;
 module.exports.CHANNELS = CHANNELS;
 module.exports.handleEventBySessionId = handleEventBySessionId;
 module.exports.handleEventByUserChannelId = handleEventByUserChannelId;
-module.exports.getSessionContext = getSessionContext;
-module.exports.initializeDb = initializeDb;
```

Comment:
Breaking change should be documented in pull request

Reply:
Agreed